### DISCUSSION TITLE
32-bit instruction set x86 support for Windows

### DISCUSSION THREAD

**Discussion by CometManAtGitHub** [serizba/cppflow#181] [General]
Is it possible to compile cppFlow for 32-bit processors?

See also: https://github.com/tensorflow/tensorflow/issues/55257

**Comment by serizba** [comment]
You first would need to have the 32-bit version of the C API. Something that it is not trivial at all, and a headaches source. I bet the code may not compile and you will need to change the sources of TensorFlow. 

If you manage to do all this, then you won't have much problem in porting cppflow.
